_rules/role-attribute-valid-value-674b10.md

```diff
@@ -42,19 +42,19 @@ Each test target has at least one token which is a valid value corresponding to
 
 ## Assumptions
 
-- This rule assumes that the `role` attribute is used to provide an ARIA [semantic role][] to the elements. If it is used for other purposes, this rule shouldn't be used.
-- This rule assumes that elements with a `role` attribute have their intended structure and relationship conveyed through some sort of presentation. If it is not the case, it is possible to fail this rule while still satisfying [Success Criterion 1.3.1 Info and Relationship][sc131].
-- This rule assumes that the intended role of the element is not its [implicit role][]. If no token is valid, User Agents will default to the [implicit role][] for the element; if that role is the intended one, it is possible to fail this rule but still satisfy [Success Criterion 1.3.1 Info and Relationships][sc131].
+There are no assumptions.
```

Comment:
I'm not sure what the discussions were so not marking it as changes requested you but I'd rather leave the first paragraph of the Assumptions section as is. As pointed out by @Jym77 some time ago `role` attribute may have some other use cases. The [Role Attribute spec](https://www.w3.org/TR/role-attribute/) says the following 

> Use cases include accessibility, device adaptation, server-side processing, and complex data description
...

> The role attribute is necessary to support Accessible Rich Internet Applications (WAI-ARIA) ... Although this is the reason the Role Attribute is published by the Protocols and Formats Working Group, the attribute has more general use cases as well.

Reply:
The reason we're moving this is because these rules no longer explicitly fail WCAG:
https://github.com/act-rules/act-rules.github.io/pull/2006/files#diff-b2c6c65374b4ddd9a243e403522e4c9237f44c8ef0c7bc5e5d601b1b12b023f6R18-R21


In ACT RF 1.1 we've introduced the concept of secondary requirements. These are requirements that can, but don't always fail the success criterion. So instead assuming that these things are never true, we're in the background explaining that if these things are true, the rule fails SC 1.3.1, but regardless if it is or isn't true, this is invalid ARIA. Hope that makes sense, if not happy to jump on a call to talk through this. This is important for the direction of ACT, so I want to get a broad perspective on this.